bin/build-cored-release

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+set -e
+
+usage() {
+  echo "Usage: $0 <releaseRef> <outputDir>"
+  echo
+  echo "Envvars:"
+  echo "GOOS:   target OS"
+  echo "GOARCH: target CPU architecture"
+  echo "DATE:   build date; defaults to now"
+  exit 1
+}
+
+trap usage ERR
+
+if [ -z "$1" -o -z "$2" ]; then
+  usage
+fi
+
+releaseRef=$1
+cd $2; outputDir=`pwd` # verify output dir and expand to absolute
+
+echo "Building ref $releaseRef to output directory $outputDir..."
+
+# Clone repository into tempdir and check out specified release ref.
+buildGoPath=`mktemp -d`
+trap "rm -rf $buildGoPath" EXIT
+
+newChain=$buildGoPath/src/chain
+mkdir -p $newChain
+git clone $CHAIN $newChain
+cd $newChain
+git checkout $releaseRef
+
+# Cross-compilation args for cored and corectl
```

Comment:
Oh, also we've been disabling CGO in scripts building `cored`. Not sure if this has been resolved. Any insight @kr?
```
# Disable cgo entirely.
# We have seen one too many problems with cgo,
# including an occasional (rare) malloc failure.
# See https://github.com/lib/pq/issues/395.
export CGO_ENABLED=0
```